maven_lambda/test/test_metadata.py

```diff
@@ -302,6 +302,12 @@ def test_generate_last_updated():
         'some/other/path':'65.0',
     },
     '65.0',
+), (
+    {
+        'android/components/with/gecko/pattern':'109.0b1',
+        'another/android/components/with/gecko/pattern':'109.0b2',
```

Comment:
Should we mix and match here and have a test case with both old and new patterns?

Reply:
Great call! This case actually uncovered a bug that would have bitten us 3 weeks from now. Fixed!